site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -155,6 +155,65 @@
             </div>
 
 
+            <div id="manual-grading" class="customization_item">
+                <h2>Manual Grading</h2>
+                <p class="rg_info_message">You may manually assign final grades to specific students. The note is only invisible to the instructor.</p>
+                <input type="hidden" name="csrf_token" value="{{ csrfToken }}" /> <!-- I'm not confident I know what this does -->
+                <input type="hidden" name="option" value="-1" /> <!-- I'm not confident I know what this does -->
+                <div class="option">
+                    <label for="manual_grading_user_id">
+                        <span>USER ID:</span>
+                        <input class="option-input" type="text" name="manual_grading_user_id" id="manual_grading_user_id">
+                    </label>
+                    <label for="manual_grading_grade">
+                        <span>Final Grade:</span>
+                        <select name="manual_grading_grade" id="manual_grading_grade">
+                            <option disabled selected value> -- select an option -- </option>
+                            <option value="A">A</option>
+                            <option value="A-">A-</option>
+                            <option value="B+">B+</option>
+                            <option value="B">B</option>
+                            <option value="B-">B-</option>
+                            <option value="C+">C+</option>
+                            <option value="C">C</option>
+                            <option value="C-">C-</option>
+                            <option value="D+">D+</option>
+                            <option value="D">D</option>
+                            <option value="F">F</option>
+                        </select>
+                    </label>
+                    <label for="manual_grading_note">
+                        <span>Note:</span>
```

Comment:
note shouldn't appear if "instructor_notes" isn't selected.

Reply:
As discussed in meeting, instead of toggling visibility of the note, there will be a message letting the professor know that the note will only appear if instructor_notes is selected. This may be changed in the future. [d1fa086](https://github.com/Submitty/Submitty/pull/10653/commits/d1fa08633957b59ed11c5710a0d048ffe896eabe)